tutorials/optimize/opt_gemm.py

```diff
@@ -25,8 +25,8 @@
 `repo <https://github.com/flame/how-to-optimize-gemm>`_. Some of them have been applied by TVM
 abstraction automatically, but some of them cannot be simply applied due to TVM constraints.
 
-All the experiment results mentioned below, are executed on 2013's 15' MacBook equiped with
-Intel i7-2760QM CPU. The cache line size should be 64 bytes for all the x86 CPU.
+All the experiment results mentioned below, are executed on 2015's 15' MacBook equiped with
+Intel i7-4770QH CPU. The cache line size should be 64 bytes for all the x86 CPU.
```

Comment:
i7-4770HQ?

Reply:
Also, in the beginning, we may want to highlight the final improvement we made based on the naive code. We should also emphasize that with the help of TVM, it was achieved by only a few lines of Python code.

Reply:
And perhaps add your name to the author list.

Reply:
Changed to i7-4770HQ